packages/fluentui/docs/src/examples/components/Input/Variations/InputExampleTargeting.shorthand.tsx

```diff
@@ -20,16 +20,22 @@ const propsTargettingInput = {
 const InputExampleTargeting = () => (
   <Grid columns="1fr 1fr" styles={{ justifyItems: 'start', alignItems: 'center', gap: '10px' }}>
     <Text content="Input with props that will be applied to either the input or the wrapper:" />
-    <Input {...propsForInput} {...propsForWrapper} />
+    <Input id="inputID-1" {...propsForInput} {...propsForWrapper} />
 
     <Text content="Input with input slot props that have to be applied to the input element:" />
-    <Input {...propsForInput} {...propsForWrapper} input={propsTargettingInput} />
+    <Input id="inputID-2" {...propsForInput} {...propsForWrapper} input={propsTargettingInput} />
 
     <Text content="Input with wrapper slot props that have to be applied to the wrapper element:" />
-    <Input {...propsForInput} {...propsForWrapper} wrapper={propsTargettingWrapper} />
+    <Input id="inputID-3" {...propsForInput} {...propsForWrapper} wrapper={propsTargettingWrapper} />
 
     <Text content="Input with input and wrapper slot props that have to be applied to the input and wrapper elements, respectively:" />
-    <Input {...propsForInput} {...propsForWrapper} input={propsTargettingInput} wrapper={propsTargettingWrapper} />
+    <Input
+      id="inputID-4"
```

Comment:
I think this will get overriden by `propsTargettingWrapper.id`.

Reply:
Nop, it's not